components/deferred-billing-terms.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+const AGE_BRACKET = 16;
```

Comment:
I wonder if AGE_BRACKET is the correct naming here.
To me, bracket suggests an interval, from / to.

Would this be a better match?
```suggestion
const CONSENT_AGE = 16;
```